Snippets/ASB/ASB_8/AddressingStrategies/NamespacePartitioningExtension.cs

```diff
@@ -0,0 +1,15 @@
+using NServiceBus;
+using NServiceBus.Configuration.AdvanceExtensibility;
+
+#region custom-namespace-partitioning-strategy-extension
+
+public static class AzureServiceBusNamespacePartitioningStrategyExtensions
+{
+    public static AzureServiceBusNamespacePartitioningSettings MySetting(this AzureServiceBusNamespacePartitioningSettings extensionPoint)
```

Comment:
i would say there is no value in returning `AzureServiceBusNamespacePartitioningSettings `